From aa9cdd87f790d71093b0b458991ac19c6ee7f7fc Mon Sep 17 00:00:00 2001
From: Martin Herkt <lachs0r@srsfckn.biz>
Date: Sat, 7 Sep 2013 22:56:00 +0200
Subject: [PATCH 1/2] Add Libs.private to openal.pc

---
 CMakeLists.txt | 6 ++++++
 openal.pc.in   | 1 +
 2 files changed, 7 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 27ec720..dcfb4ad 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -913,6 +913,12 @@ ENDIF()
 IF(LIBTYPE STREQUAL "STATIC")
     ADD_DEFINITIONS(-DAL_LIBTYPE_STATIC)
     SET(PKG_CONFIG_CFLAGS -DAL_LIBTYPE_STATIC ${PKG_CONFIG_CFLAGS})
+
+    FOREACH(l ${EXTRA_LIBS})
+        IF(NOT l MATCHES "^-.*")
+            SET(PKG_CONFIG_LIBS_PRIVATE "${PKG_CONFIG_LIBS_PRIVATE} -l${l}")
+        ENDIF()
+    ENDFOREACH()
 ENDIF()
 
 # Needed for openal.pc.in
diff --git a/openal.pc.in b/openal.pc.in
index 8bdd4f3..d8759d0 100644
--- a/openal.pc.in
+++ b/openal.pc.in
@@ -8,4 +8,5 @@ Description: OpenAL is a cross-platform 3D audio API
 Requires: @PKG_CONFIG_REQUIRES@
 Version: @PACKAGE_VERSION@
 Libs: -L${libdir} -l@LIBNAME@ @PKG_CONFIG_LIBS@
+Libs.private: @PKG_CONFIG_LIBS_PRIVATE@
 Cflags: -I${includedir} -I${includedir}/AL @PKG_CONFIG_CFLAGS@
-- 
1.9.1

